tests/UI/campaigns/commonTests/loginFO.js

```diff
@@ -0,0 +1,24 @@
+require('module-alias/register');
+const {expect} = require('chai');
+const testContext = require('@utils/testContext');
+const loginPage = require('@pages/FO/login');
+const myAccountPage = require('@pages/FO/myAccount');
+
+module.exports = {
+  async loginFO(mochaContext, page, customer) {
```

Comment:
If We add common tests about FO, we should update one checked identifiers `tools/stepIdentifiersChecker.js`.